resources/views/platform/components/game-list-item/suggest-reason.blade.php

```diff
@@ -0,0 +1,61 @@
+@props([
+    'selectionMethod' => 'random',
+    'relatedSubject' => '',
+    'relatedGameId' => 0,
+    'relatedGameTitle' => '',
+    'relatedGameIcon' => '',
+])
+
+@if ($selectionMethod == 'want-to-play')
+    From your Want to Play list
+@elseif ($selectionMethod == 'similar-to')
+    @if ($relatedGameId > 0)
+        Similar to mastered game
+        <x-game.avatar
+            :gameId="$relatedGameId"
+            :gameTitle="$relatedGameTitle"
+            :gameImageIcon="$relatedGameIcon"
+            :iconSize="16"
+        />
+    @else
+        In Similar Games collection
+    @endif
+@elseif ($selectionMethod == 'common-hub')
+    Shares {{ $relatedSubject }}
+    @if ($relatedGameId > 0)
+        with mastered game
+        <x-game.avatar
+            :gameId="$relatedGameId"
+            :gameTitle="$relatedGameTitle"
+            :gameImageIcon="$relatedGameIcon"
+            :iconSize="16"
+        />
+    @endif
+@elseif ($selectionMethod == 'common-author')
+    Shares set developer {!! userAvatar($relatedSubject, icon: false) !!}
```

Comment:
Similar feedback to naming the hub. Letting the user know a recommendation is due to a common author seems very valuable, but I'm not so sure most users will initially be interested in the name of the dev (which is also present at the top of the game page).

I feel this should simply read "Shares set developer with mastered game:".